src/main/groovy/org/labkey/gradle/util/BuildUtils.groovy

```diff
@@ -39,10 +39,10 @@ class BuildUtils
     public static final String SERVER_MODULES_DIR = "server/modules"
     public static final String PLATFORM_MODULES_DIR = "server/modules/platform"
     public static final String COMMON_ASSAYS_MODULES_DIR = "server/modules/commonAssays"
-    public static final String CUSTOM_MODULES_DIR = "server/customModules"
+    public static final String CUSTOM_MODULES_DIR = "server/customModules" // TODO remove after 20.6
```

Comment:
Can't this be removed now? This is not a real module place, AFAIK...

Reply:
Sure. Done